Update Dockerfile

StevenBOULLE opened the pull request:
Change rights to 775

<!-- Thanks for sending us a PR to improve this project! If you are adding a 
feature or fixing a bug, and this needs more documentation, please add it to your PR. -->

**Thanks for contributing to the Docker-Selenium project!**
**A PR well described will help maintainers to quickly review and merge it**

Before submitting your PR, please check our [contributing](https://selenium.dev/documentation/en/contributing/) guidelines, applied for this repository.
Avoid large PRs, help reviewers by making them as simple and short as possible.


<!--- Provide a general summary of your changes in the Title above -->

### Description

### Motivation and Context

### Types of changes
<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Checklist
<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
- [x] I have read the [contributing](https://selenium.dev/documentation/en/contributing/) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
<!--- Provide a general summary of your changes in the Title above -->

diemol committed:
Tuning chmod permissions

diemol committed:
Tuning chmod permissions

diemol committed:
Kick CI

diemol committed:
More tuning

diemol committed:
More tuning

diemol committed:
Tuning a bit more

diemol committed:
Tuning a bit more

StevenBOULLE committed:
Update Dockerfile

Change rights to 775

StevenBOULLE committed:
Update Dockerfile

Change rights

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/signed)](https://cla-assistant.io/SeleniumHQ/docker-selenium?pullRequest=1773) <br/>All committers have signed the CLA.

diemol committed:
Merge branch 'update-chmod-permissions' into patch-2

diemol commented:
Thanks for the PR, @StevenBOULLE! Sadly it still fails.

StevenBOULLE committed:
Change group for some directories

Change group of directories "/opt/selenium ${HOME} /opt/selenium/assets /var/run/supervisor /var/log/supervisor" to group seluser

diemol commented on Base/Dockerfile:
```diff
@@ -70,12 +70,12 @@ COPY supervisord.conf /etc
   && wget --no-verbose https://repo1.maven.org/maven2/org/seleniumhq/selenium/selenium-http-jdk-client/4.7.2/selenium-http-jdk-client-4.7.2.jar \
     -O /opt/selenium/selenium-http-jdk-client.jar \
-  && chgrp -R 0 /opt/selenium ${HOME} /opt/selenium/assets /var/run/supervisor /var/log/supervisor \
+  && chgrp -R 1201 /opt/selenium ${HOME} /opt/selenium/assets /var/run/supervisor /var/log/supervisor \
```
I believe this would break when using the images in OpenShift, https://docs.openshift.com/container-platform/3.11/creating_images/guidelines.html#openshift-specific-guidelines

StevenBOULLE committed:
Usage of ACL

Usage of ACL to stay with root's rights for user and group but add some rights to user seluser with ACL

StevenBOULLE committed:
Install ACL package

diemol commented on Base/Dockerfile:
```diff
@@ -70,13 +71,14 @@ COPY supervisord.conf /etc
   && chgrp -R 0 /opt/selenium ${HOME} /opt/selenium/assets /var/run/supervisor /var/log/supervisor \
-  && chmod -R g=u /opt/selenium ${HOME} /opt/selenium/assets /var/run/supervisor /var/log/supervisor
+  && chmod -R g=u /opt/selenium ${HOME} /opt/selenium/assets /var/run/supervisor /var/log/supervisor \
+  && setfacl -mR u:seluser:rwx file /opt/selenium ${HOME} /opt/selenium/assets /var/run/supervisor /var/log/supervisor
```
Could you please add a comment explaining what this is doing?

StevenBOULLE replied:
ACLs are used beside of Unix permissions to grant more precise rights.
In my command, I give the user seluser RWX rights recursively on all the directories listed

StevenBOULLE committed:
Merge branch 'trunk' into patch-2

StevenBOULLE committed:
typo in command setfacl

StevenBOULLE committed:
change stefacl location on script

StevenBOULLE reviewed (commented):
explanation given

StevenBOULLE committed:
Change setfacl location once again

StevenBOULLE committed:
add acl to /opt

StevenBOULLE commented:
I have 2 merge requests, I close one

StevenBOULLE commented:
missunderstanding

StevenBOULLE committed:
Update Dockerfile

Adding sudo to avoid "permission denied" messages

StevenBOULLE committed:
Merge branch 'trunk' into patch-2

StevenBOULLE committed:
Merge branch 'trunk' into patch-2

diemol reviewed (approved):
Thank you, @StevenBOULLE!
